apps/test/unit/lib/kits/maker/ui/MakerSetupGuideTest.js

```diff
@@ -0,0 +1,66 @@
+/** @file Test SetupGuide component */
+import React from 'react';
+import {expect} from '../../../../../util/deprecatedChai';
+import {shallow} from 'enzyme';
+import MakerSetupGuide from '@cdo/apps/lib/kits/maker/ui/MakerSetupGuide';
+import applabI18n from '@cdo/applab/locale';
+import experiments from '@cdo/apps/util/experiments';
+
+describe('MakerSetupGuide', () => {
+  describe('Microbit experiment is not enabled', () => {
+    let description;
+    let markdown;
+    it('renders circuit playground description', () => {
+      const wrapper = shallow(<MakerSetupGuide />);
+      description = wrapper.findWhere(
```

Comment:
I would just place the `let` here instead of outside the test's `it` body. (tho I'd also use a `find` and not a `findWhere`... see my other comment)

```
let description = wrapper.findWhere(
```